dfastmi/io.py

```diff
@@ -1381,9 +808,25 @@ def read_xyc(
             colnames = ["Val", "X", "Y"]
         else:
             colnames = ["X", "Y"]
-        P = pandas.read_csv(
-            filename, names=colnames, skipinitialspace=True, delim_whitespace=True
-        )
+        if hasHeader:
```

Comment:
Are these functional changes to this method?

Reply:
Discussed, this is needed for read_xyc to function with a correct file instead of a user changed file.